Products/CMFCore/tests/test_ActionInformation.py

```diff
@@ -175,6 +175,9 @@ def __getitem__(self, k):
     def __len__(self):
         return len(self._data)
 
+    def getVirtualRoot(self):
+        return '/'
```

Comment:
I don't really have a context for this fix, is this hardcoded value a safe choice?

Reply:
The failing test (see https://travis-ci.org/github/zopefoundation/Products.CMFCore/jobs/666271314#L552-L612) renders the following template line: 
https://github.com/zopefoundation/Zope/blob/06ee93caa3226d26e6f4b12681bc6b0269e10462/src/App/dtml/manage_navbar.dtml#L13 and breaks there because the used `DummyRequest` does not have a `getVirtualRoot` method. The value returned by the method is irrelevant for the test as the rendering happens outside the area of interest for the test.